### PR TITLE
NCL-2293 ssh credentials moved to rest api

### PR DESCRIPTION
**Pull request by michalszynkiewicz**

### CODE REVIEW

File: openshift-environment-driver/src/main/java/org/jboss/pnc/environment/openshift/OpenshiftStartedEnvironment.java [hunk 1]
```diff
@@ -170,8 +170,7 @@ private void initDebug() {
 
             debugData.setSshServiceInitializer(d -> {
                 Integer port = startSshService();
-                d.setSshPort(port);
-                d.setSshHost(route.getHost());
+                d.setSshCommand("ssh worker@" + route.getHost() + " -p " + port);
```
Comment (pgier) [inline openshift-environment-driver/src/main/java/org/jboss/pnc/environment/openshift/OpenshiftStartedEnvironment.java]: Maybe should use a final String for the username "worker".

**Comment by pgier** [comment]
Code looks fine to me.  Would it be better to just always return the ssh credentials in the BuildRecord even if the current user is not the one who started the build?

**Comment by michalszynkiewicz** [comment]
@pgier it was decided that it would be unsafe. For now we're showing credentials for everybody, but later we want to hide it...

File: openshift-environment-driver/src/main/java/org/jboss/pnc/environment/openshift/OpenshiftStartedEnvironment.java (hunk 1 above)
Comment (michalszynkiewicz) [inline openshift-environment-driver/src/main/java/org/jboss/pnc/environment/openshift/OpenshiftStartedEnvironment.java]: done

**Pull request merged**